Directory.Build.props

```diff
@@ -4,9 +4,15 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
     <PackageReference Include="OpenCover" Version="4.6.519" PrivateAssets="All" />
     <PackageReference Include="ReportGenerator" Version="3.1.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" PrivateAssets="All" />
```

Comment:
I've started parameterising the version in other repos because they update in sync, so it's only one thing to change, rather than 4.

Reply:
I'm not such a fan of that abstraction, as it makes life harder for any build tools that might try to red and modify it.

Reply:
Up to you, but it's a pattern with MS precedent (kind of where I got the idea to use it myself): https://github.com/aspnet/HttpClientFactory/blob/master/build/dependencies.props